Jenkinsfile

```
@@ -15,7 +15,7 @@ node {
           echo "Running a subset of the tests to check the content schema changes"
           govuk.runRakeTask("test:publishing_schemas --trace")
         } else {
-          sh("bundle exec rake")
+          sh("bundle exec rake lint test cucumber jasmine")
```

Hmmm I understand why you haven't just removed `pact:verify` from the default task (devs should be able to run the full test suite locally), but I think it would be cleaner to define a new Rake task that encapsulates the above, e.g.

```ruby
task ci: %i[lint test cucumber jasmine] # Pact tests run separately on GitHub Actions
```

And then changing this to:

```suggestion
          sh("bundle exec rake ci")
```

But not a blocker.

---

Sure, this could work but may need some refinement.

I don't like the idea of it being `ci` as pact:verify will be also a part of CI. Could call it `jenkins_ci`, but that couples it to Jenkins. I suppose could be `ci_build` hmmm. I also wonder if it's actually better to be direct in the Jenkinsfile and whether adding another task ends up being misdirection.

---

Fair point 👍 happy to leave as-is.